### PR TITLE
Added a pair of snippets for creating and referencing anchors (block IDs)

### PR DESCRIPTION
**Pull request by polyglot-jones**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,8 +108,31 @@ For the above reasons, it's possible that during the Alpha stage various syntax
...
+| Anchor Reference   | `<<` <kbd>Tab</kbd>                       |
 
 
+### Anchored Subsection
+
+This snippet ensures that your anchor names properly conform (lower-case, no spaces).
+
+Type this: `[[` <kbd>Tab</kbd> Navigable Text
+
+And you get this:
```
Comment (tajmone) [inline README.md:121]: Add an empty line before the ` ``` ` block delimiters!

File: README.md [hunk 2]
```diff
@@ -108,8 +108,31 @@ For the above reasons, it's possible that during the Alpha stage various syntax
...
+```
+    [[navigable-text]]
```
Comment (tajmone) [inline README.md:123]: Shouldn't we remove the indentation here?

**Review by tajmone — COMMENTED** [review]
Since I've merged two of your other PRs, you'll have to rebase again your PR branch, due to same files being edited. But it should be a smooth operation, without conflicts.

**Comment by polyglot-jones** [comment]
No idea why I can't get this to rebase. I'm closing the PR and will start over.

**Pull request closed without merging**